### PR TITLE
feat: simple callback

### LINKED ISSUE #63: Implement callback functionality

### PR DESCRIPTION
**Pull request by Vovke**
Resolves #63

### CODE REVIEW

File: src/state.rs [hunk 1]
```diff
@@ -132,7 +132,17 @@ impl State {
                                 // Only perform actions if the record is saved in ledger
                                 match state.db.mark_paid(id.clone()).await {
                                     Ok(order) => {
-                                        // TODO: callback here
+                                        if let Some(callback) = order.callback.clone() {
+                                            tokio::spawn(async move {
+                                                tracing::info!("Sending callback to: {}", callback);
+                                                let client = reqwest::Client::new();
```
Comment (Slesarew) [inline src/state.rs:138]: is this like 3rd different entity called just "client" here?

Comment (Vovke) [reply src/state.rs]: addressed in 4576c71

File: Cargo.toml [hunk 2]
```diff
@@ -63,6 +63,7 @@ substrate_parser = { git = "https://github.com/Alzymologist/substrate-parser" }
 substrate-constructor = { git = "https://github.com/Alzymologist/substrate-constructor" }
 mnemonic-external = { git = "https://github.com/Alzymologist/mnemonic-external" }
 substrate-crypto-light = { git = "https://github.com/Alzymologist/substrate-crypto-light" }
+reqwest = "0.12.8"
```
Comment (fluiderson) [inline Cargo.toml:66]:
```suggestion
reqwest = "0.12"
```
Let's specify only the latest major version. It does the same & looks conciser.

**Review by fluiderson — COMMENTED** [review]
https://github.com/Alzymologist/Kalatori-backend/blob/56cda51716ee0d11491b031838503380047f6fa3/Cargo.toml#L23
Remove this HTTP client crate from deps if you decided to use `reqwest` instead.

File: Cargo.toml (hunk 2 above)
Comment (Vovke) [reply Cargo.toml:65]: addressed in 4cc2713

**Review by fluiderson — APPROVED** [review]

**Comment by Vovke** [comment]
included in #106

**Pull request closed without merging**